frontend/src/employee-mobile-frontend/common/TopBar.tsx

```diff
@@ -68,40 +74,87 @@ export default React.memo(function TopBar({
 }: Props) {
   const { i18n } = useTranslation()
 
+  const notificationResult = useQueryResult(currentSystemNotificationQuery())
+
+  // only shown on main screen
+  const hideSystemNotification = invertedColors
+
+  const [showNotificationModal, setShowNotificationModal] = useState(false)
```

Comment:
`useBoolean` would give you on/off functions for free

Reply:
At least when there's no need for fancier things like `toggle`, I personally find the standard hook more readable and with less naming to think about, and `useBoolean` seems like an over-abstraction. 🤔  For example here the name for the `on`-function would naturally be `showNotificationModal`, but then what should the boolean value be called?